webportal/package.json

```diff
@@ -37,6 +37,7 @@
     "express": "4.16.2",
     "extract-text-webpack-plugin": "3.0.2",
     "file-loader": "1.1.5",
+    "floatthead": "2.0.3",
```

Comment:
Please update `yarn.lock` file accordingly. [Install yarn](https://yarnpkg.com/en/docs/install#windows-tab) and then run: `npm run yarn install`.

Reply:
Solved